lib/commandCenter/httpBatchHandler.js

```diff
@@ -243,6 +243,13 @@ function parseMultipart(req, boundary, cb) {
 	// pipe all incoming data straight into the multipart parser
 
 	req.pipe(parser);
+
+	req.on('error', function (error) {
+		delete cmdData;
+		delete files;
```

Comment:
These two lines do nothing. `delete` is an operator to remove properties from objects.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete

Reply:
So I guess the data will just naturally expire in this case? Are there any cleanup steps we should actively take?

Reply:
I don't think you would have to do anything. The garbage collector should just clean it up. If it didn't, we would have a leak which would be a bigger problem regardless.

Reply:
Right. Perhaps my question was more about whether we can clean up the data so that it doesn't have to wait for garbage collection, but I am guessing this isn't possible?